pyproject.toml

```diff
@@ -57,11 +57,6 @@ extra-dependencies = [
   "transformers>=4.47.0, <4.48.0", # https://github.com/huggingface/transformers/issues/35639
   # RAG evaluation harness (SAS evaluator)
   "sentence-transformers>=3.0.0",
-  # Tools support
-  "jsonschema",
```

Comment:
this change is unrelated but starting from 2.10.0, this dependency comes with Haystack